.github/ISSUE_TEMPLATE/2-feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "\U0001F680 Feature request"
+about: Suggest an idea for this project
+
+---
+
+<!--
+Thank you for suggesting an idea to make Apollo Voyager Server better.
+
+Please fill in as much of the template below as you're able.
+-->
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
Is this template appears for who click on in the button? Is GitHub able to work with more than one issue template?